packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -414,8 +414,14 @@ class IOSSimulator extends Device {
     return false;
   }
 
+  String _simulatorHomePath;
+  String get simulatorHomePath {
+     _simulatorHomePath ??= platform.environment['IOS_SIMULATOR_HOME'] ?? homeDirPath;
+    return _simulatorHomePath;
+  }
```

Comment:
Would it be better to have a `simulatorLogPath` and pass the full directory? That would allow us to avoid hardcoding the assumptions around the relative path of the log below.

Reply:
Done.  It has to support the simulator id, so I made it substitute `%{id}`